src/client/windshaft.js

```diff
@@ -9,6 +9,8 @@ const SAMPLE_ROWS = 1000;
 const MIN_FILTERING = 2000000;
 const REQUEST_GET_MAX_URL_LENGTH = 2048;
 
+export const CLUSTER_FEATURE_COUNT = '_cdb_feature_count';
```

Comment:
For consistency and to keep track of every ID and pattern like this, this should be on `schema.js`